### PR TITLE
Fix menu item case

### LINKED ISSUE #503: Some menu items don't follow sentence case convention
Autolab uses sentence case (first word capitiized, following words lower case) for menu items in the "Admin options", "Admin options (CA)", and "Options" lists. However, there are some list items that don't following this convention (e.g., Autograder Settings, Scoreboard Settings, Download Section Submissions, View Scoreboard), which creates a dissonance for the reader. 

All list items need to be changed to follow sentence case.

### PR DESCRIPTION
**Pull request by cjford**
Fixes #503

### CODE REVIEW

File: app/views/assessments/show.html.erb [hunk 1]
```diff
@@ -65,7 +65,7 @@
     <% if @cud.course_assistant? or @cud.instructor? then %>
         <h3>Admin Options (CA)</h3>
         <ul class="options">
-          <li><%= link_to "Grade section #{@cud.section} submissions", url_for(:action => 'viewGradesheet', :section => '1', :escape => false), 
+          <li><%= link_to "Grade section #{@cud.section.downcase} submissions", url_for(:action => 'viewGradesheet', :section => '1', :escape => false),
```
Comment (jez) [inline app/views/assessments/show.html.erb]: We want to keep `@cud.secion` uppercase, actually. This is usually something that people will expect to be uppercase, like a single letter (Section H), or more generally some name (think: proper noun) for the section.

File: app/views/assessments/show.html.erb [hunk 2]
```diff
@@ -75,7 +75,7 @@
             :title=> "Reload the assessment configuration file (provided for backward compatibility with legacy assessments)" %></li>
 
           <% if !@assessment.disable_handins %>
-            <li><%= link_to  "Download Section #{@cud.section} Submissions", [:downloadAll, @course, @assessment, :submissions, final: true], title: "Download Section #{@cud.section} Submissions" %></li>
+            <li><%= link_to  "Download section #{@cud.section.downcase} submissions", [:downloadAll, @course, @assessment, :submissions, final: true], title: "Download section #{@cud.section.downcase} submissions" %></li>
```
Comment (jez) [inline app/views/assessments/show.html.erb]: [Ditto][1]

[1]: https://github.com/autolab/Autolab/pull/535/files#r34955560

**Comment by jez** [comment]
Thanks! This look awesome.

There's a problem with your PR though (in addition to the inline comments I left): your commit is branched off of `master`, but you're requesting a pull into `develop`. You should rebase your feature branch on top of `develop` and force push the change:

```
git checkout fix_menu_item_case
git rebase --onto origin/develop 'HEAD^'
git push --force origin fix_menu_item_case
```

**Comment by jez** [comment]
@ymzong Do you know why the Travis build is failing? The error seems to be a malformed cURL command?

**Comment by cjford** [comment]
Got it, thanks. I'll fix it up and try again. 

It might be worth adding a step to rebase on develop in the contributing guide, since the step by step leaves you with a branch off master. Probably common sense, but it could help out us newbies.

**Comment by jez** [comment]
Good call on the CONTRIBUTING.md. I've updated it appropriately. Your PR is also merged now, so it should show up in the next release of Autolab.

**Pull request merged**